lib/libc/picolibc/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_library()
+zephyr_library_sources(libc-hooks.c)
+
+if(CONFIG_PICOLIBC AND NOT CONFIG_PICOLIBC_USE_MODULE)
```

Comment:
`if(CONFIG_PICOLIBC` is redundant because the `picolibc` subdirectory is only added when `CONFIG_PICOLIBC=y`.

Reply:
Fixed.

Reply:
It looks like this change was not pushed.